### DISCUSSION TITLE
manageDeclaretionFinalizeRequest/ preliminatyConfirmation

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#44] [Kérdések és válaszok - Gépi interfész]
Kedves NAV szakértők!

A manageDeclaretionFinalizeRequest/ preliminatyConfirmation =  false elem mire szolgál?

A dokumentációban ennyit találtunk róla: "A bevallás előzetes jóváhagyása, ha abban feldolgozást blokkoló hiba nincs." És, hogy a default értéke egyelőre false kell legyen.

Van esetleg ennél több infó, ami tudni kellene ebben a fázisban?

Köszönjuk előre is!

**Comment by renced42** [comment]
A /manageDeclarationFinalize az áfaanalitika beküldésének véglegesítésére szolgáló operáció. Ezen keresztül van lehetőség jelezni, hogy a kliens oldali partíció feltöltések befejeződtek, és megkezdődhet a feldolgozás.
A FALSE érték hatására nem kezdődik meg a feldolgozás. Ezért ezt explicite meg kell mondani hogy kezdhető (TRUE).
